src/hssm/distribution_utils/dist.py

```diff
@@ -694,7 +692,13 @@ def make_likelihood_callable(
         if loglik_kind == "analytical":
             if backend is None or backend == "pytensor":
                 return loglik
-            return make_blackbox_op(loglik)
+            logp_funcs = make_jax_logp_funcs_from_callable(
```

Comment:
Let's put on the agenda for Tuesday's team meeting to go through the branching logic of this. It was confusing before and it's still a bit hairy I find.

Would be great if we could simplify / make more readable.